Source/WebCore/bindings/js/JSNodeCustom.cpp

```diff
@@ -71,7 +71,7 @@ bool JSNodeOwner::isReachableFromOpaqueRoots(JSC::Handle<JSC::Unknown> handle, v
 {
     auto& node = jsCast<JSNode*>(handle.slot()->asCell())->wrapped();
     if (!node.isConnected()) {
-        if (GCReachableRefMap::contains(node)) {
+        if (GCReachableRefMap::contains(node) || node.isInCustomElementReactionQueue()) {
```

Comment:
How about first checking `node.isInCustomElementReactionQueue()`? Given that we already read `isConnected()` bit, checking this is free. And we may skip `GCReachableRefMap` lookup.

Reply:
Oh, GCReachableRefMap::contains(node) just checks a bit flag on Node as well.